pallets/crowdloans/src/migrations.rs

```diff
@@ -0,0 +1,111 @@
+// Copyright 2021 Parallel Finance Developer.
+// This file is part of Parallel Finance.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! # Add vaults for batch 1 winning projects
+
+use super::*;
+use frame_support::traits::Get;
+use primitives::ParaId;
+use sp_runtime::traits::Zero;
+use sp_std::{vec, vec::Vec};
+use types::*;
+/// Add vaults for batch 1 winning projects
+pub fn migrate<T: Config>() -> frame_support::weights::Weight {
```

Comment:
Since we have storage migration, maybe we should have a version management

Reply:
we can call `migrations::v?::migrate` in runtime

Reply:
Good point, do you think if we need storage like this:
https://github.com/paritytech/substrate/blob/961666a2cb/frame/staking/src/pallet/mod.rs#L462
and an Enum like this:
https://github.com/paritytech/substrate/blob/961666a2cb/frame/staking/src/lib.rs#L765-L774

Reply:
Yes I think so